### ISSUE TITLE
some CardDav servers put "\," into vCard downloads of attributes

### ISSUE THREAD

**Issue by derwok** [andig/carddav2fb#90]
**TL;DR: Pull request in preparation.**

Some CardDav servers - e.g., Baikal (sabre/dav) - sends a attribute with embedded comma (,) in XML with "\\,". The quote backslash survives all our processing chain right into the Fritz!Box upload.

Example: Display in CardDav-Editor:
![grafik](https://user-images.githubusercontent.com/4393698/51073840-3b1a2800-1677-11e9-94b4-23ace1e83056.png)

Example: Display in Fritz!Box Browser UI:
![grafik](https://user-images.githubusercontent.com/4393698/51073933-05297380-1678-11e9-8cb3-e69d0b13c688.png)

Root cause is the specified BACKSLASH quoting of COMMA, SEMICOLON & BACKSLASH described in the vCard standard:
https://tools.ietf.org/html/rfc6350#section-3.4

As a consequence, FritzFon displays ugly "\\" in some contacts names. And even breaks Phone numbers with comma. (A comma in a phone number is a pause!)

Solution: replace \\{SPECIALCHAR} => {SPECIALCHAR} in values in the vCard parser
